configs/_base_/models/fcn_unet_s5-d16.py

```diff
@@ -39,6 +39,7 @@
         num_classes=2,
         norm_cfg=norm_cfg,
         align_corners=False,
+        ignore_index = 255,
```

Comment:
I think this line is unnecessary in that it has been set to `255` at default in [decode_head](https://github.com/open-mmlab/mmsegmentation/blob/dev-1.x/mmseg/models/decode_heads/decode_head.py#L101).